content/python/concepts/built-in-functions/terms/globals/globals.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'globals()'
+Description: 'Returns a dictionary with all the global variables and symbols for the current program.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Dictionary'
+  - 'Scope'
+  - 'Symbol'
+  - 'Values'
+  - 'Variable Types'
+  - 'Variables'
+ CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The built-in **globals()** function allows you to access the global scope's name table, which is a writable dictionary containing the current global names and their corresponding values in your code. You can also use this function to access or even modify the value of a global variable from with functions.
```

Comment:
```suggestion
The built-in **globals()** function allows access to the global scope's name table, which is a writable dictionary containing the current global names and their corresponding values in the code. This function can be used to access or modify the value of a global variable from within functions.
```
Try not to use pronouns.

Reply:
In addition to avoiding first- and second- person pronouns, put single backticks around `globals()` as well as bolding it:

```
**`globals()`**
```